docs/index.md

```diff
@@ -130,6 +130,7 @@ auto_examples/snowflake_agent/index
 auto_examples/k8s_spark_plugin/index
 auto_examples/sql_plugin/index
 auto_examples/kftensorflow_plugin/index
+auto_examples/wandb_plugin/index
```

Comment:
@cosmicBboy From the [CI issue](https://productionresultssa7.blob.core.windows.net/actions-results/9926b81f-136a-498b-8003-f0cb11bfec12/workflow-job-run-7e620c85-24a8-5ffa-8b1f-642bc9b1fc36/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-05-10T17%3A51%3A22Z&sig=wiCIb30toGVdBr3KL54xsVTdyoBawKv8INBoFxKC7kg%3D&sp=r&spr=https&sr=b&st=2024-05-10T17%3A41%3A17Z&sv=2021-12-02) is there another place I need to add `auto_examples/wandb_plugin/index` to the toc?

Reply:
ah, so this `index.md` page actually isn't used any more (we still haven't sunsetted the old docs @ppiegaze @neverett, we need to decide when we want to do this). it's fine if it's here, but the entry needs to be added somewhere on this page: https://github.com/flyteorg/flytesnacks/blob/master/docs/integrations.md in the appropriate `list-table` directive.

**For context:**
The sphinx dark magic that's happening here is, these list tables are being picked up in the flyte docs [here](https://github.com/flyteorg/flyte/blob/master/docs/conf.py#L356-L359) and converted into `{list-table-toc}`s that automatically add them to the flyte monodocs toc: https://github.com/flyteorg/flyte/blob/master/docs/_ext/import_projects.py#L59-L81